File: base_java/src/main/java/com/hesheng1024/base_java/ui/IBaseView.java
package com.hesheng1024.base_java.ui;

import androidx.annotation.NonNull;

/**
 * @author hesheng1024
 * @date 2019/5/22 15:10
 */
public interface IBaseView {

    /**
     * toast
     *
     * @param msg String
     */
    void toastMsg(@NonNull String msg);

    /**
     * finish
     */
    void finishActivity();

    /**
     * no more data
     */
    void noMore();
}

File: base_kotlin/src/main/java/com/hesheng1024/base_kotlin/ui/IBaseView.kt
package com.hesheng1024.base.base

import androidx.annotation.NonNull

/**
 *
 * @author hesheng1024
 * @date 2020/2/7 16:59
 */
interface IBaseView {

    fun toastMsg(@NonNull msg: String)

    fun finishActivity()

    fun noMore()
}